docs-kits/kits/Circularity_KIT/page_changelog.md

```diff
@@ -5,9 +5,25 @@ description: "Circularity KIT"
 sidebar_position: 1
 ---
 
-![Circulairty kit banner](/img/kit-icons/circularity-kit-icon.svg)
+![Circularity kit banner](/img/kit-icons/circularity-kit-icon.svg)
 All notable changes to this Kit will be documented in this file.
 
+## [1.2.0] - 15.05.2024
+
+### Added
+
+- add new figures
+- add license files for new figures
+
+### Changed
+
+- change structure of the contents. Same content chapters are in different order
+- change wording
+
+### Removed
+
+- ./.
+
 ## [1.3.0] - 17.09.2024
```

Comment:
I'm confused here about the order? Why is version 1.2.0 newer than version 1.3.0?